python/lsst/pipe/tasks/multiBand.py

```diff
@@ -111,12 +116,117 @@ def setDefaults(self):
         self.detection.isotropicGrow = True
         self.detection.background.undersampleStyle = 'REDUCE_INTERP_ORDER'
 
+## \addtogroup LSST_task_documentation
+## \{
+## \page DetectCoaddSourcesTask
+## \ref DetectCoaddSourcesTask_ "DetectCoaddSourcesTask"
+## \copybrief DetectCoaddSourcesTask
+## \}
 
 class DetectCoaddSourcesTask(CmdLineTask):
-    """Detect sources on a coadd
-
-    This operation is performed separately in each band.  The detections from
-    each band will be merged before performing the measurement stage.
+    """!
+\anchor DetectCoaddSourcesTask
+
+\brief Detect sources on a coadd
+
+\section pipe_tasks_multiBand_Contents Contents
+
+  - \ref pipe_tasks_multiBand_DetectCoaddSourcesTask_Purpose
+  - \ref pipe_tasks_multiBand_DetectCoaddSourcesTask_Initialize
+  - \ref pipe_tasks_multiBand_DetectCoaddSourcesTask_Run
+  - \ref pipe_tasks_multiBand_DetectCoaddSourcesTask_Config
+  - \ref pipe_tasks_multiBand_DetectCoaddSourcesTask_Debug
+  - \ref pipe_tasks_multiband_DetectCoaddSourcesTask_Example
+
+\section pipe_tasks_multiBand_DetectCoaddSourcesTask_Purpose	Description
+
+Command-line task that detects sources on a coadd of exposures obtained with a single filter.
+
+We want to detect sources on coadded images in prep for eventually running forced photometry consistently 
+across multiple filter bands. Two things must be achieved to finish coadd processing. Coadding individual 
+visits requires each exposure to be warped. This introduces covariance in the noise properties across pixels. 
+We correct the coadd variance by scaling the variance plane in the coadd to match the observed variance - this 
+is not the right thing to do but is simple and works well in practice. We must also detect sources and 
+generate footprints for the parent sources - this is delegated to the existing SourceDetectionTask.
+
+  \par Inputs:
+    deepCoadd{tract,patch,filter}: ExposureF
```

Comment:
Name depends on config, I think -- it's only ``deepCoadd`` if ``DetectCoaddSourcesConfig.coaddName`` is ``deep``. (Also applies to the below)